acceptance/tests/direct_puppet/static_catalog_env_control.rb

```diff
@@ -0,0 +1,348 @@
+test_name "Environment control of static catalogs" 
+require 'json'
+
+@testroot = master.tmpdir(File.basename(__FILE__, '/*'))
+### @testroot = "/etc/puppetlabs"
+
+@coderoot = "#{@testroot}/code"
+
+@master_opts = {
```

Comment:
I don't understand.  Which is unnecessary? master_opts, or main/environmentpath?
puppetserver config does not specify environmentpath.  Should it?
We should not test if not using puppetserver.  How can the test tell?